Lib/test/test_binascii.py

```diff
@@ -110,6 +110,36 @@ def addnoise(line):
         # empty strings. TBD: shouldn't it raise an exception instead ?
         self.assertEqual(binascii.a2b_base64(self.type2test(fillers)), b'')
 
+    def test_base64errors(self):
+        # Test base64 with invalid padding
+        def assertInvalidPadding(data):
+            with self.assertRaises(binascii.Error,
+                                   expected_regexp=r'(?i)Invalid padding'):
```

Comment:
I don't remember other case of passing this argument as keyword. I'm not sure the name of this parameter is the part of the public API.

And seems you meant `assertRaisesRegex`.